test/data/users_data.dart

```diff
@@ -0,0 +1,11 @@
+import 'package:built_collection/built_collection.dart';
+import 'package:crowdleague/models/auth/provider_info.dart';
+import 'package:crowdleague/models/auth/user.dart';
+
+// Create a user object for use in tests.
+User get bobUser => User(
```

Comment:
just an attempt to DRY us up a bit 🏜️

Reply:
I made it a getter so we can use `bobUser` wherever we want but it creates a new object each time. (at least I think it does... not really sure how top level getters work now I think about it...  In practice I think we're fine either way because we're using immutable built_value objects)